xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -112,7 +112,7 @@
       },
       "order": { "type": "string", "default": "ascending", "enum": [ "ascending", "descending" ] },
       "ignorearticle": { "type": "boolean", "default": false },
-      "useartistsortname": { "type": "boolean", "default": false }	  
+      "useartistsortname": { "type": "boolean", "default": false }
```

Comment:
A whitespace change to an area that you aren't otherwise touching. This makes git blame hard to read etc., so can you drop the white space clean up that your editor has done here and a few other places

Reply:
Feels wrong to readd a valid cleanup, but I agree that it makes it harder to read. Reverted.